resip/stack/ReceivedParameter.cxx

```diff
@@ -0,0 +1,77 @@
+#include "resip/stack/ReceivedParameter.hxx"
+#include "resip/stack/Symbols.hxx"
+#include "rutil/ParseBuffer.hxx"
+#include "rutil/Logger.hxx"
+
+using namespace resip;
+
+#define RESIPROCATE_SUBSYSTEM resip::Subsystem::SIP
+
+ReceivedParameter::ReceivedParameter(ParameterTypes::Type type,
+                                     ParseBuffer& pb,
+                                     const std::bitset<256>& terminators) :
+   DataParameter(type)
+{
+   pb.skipWhitespace();
+
+   if (!pb.eof() && *pb.position() == Symbols::EQUALS[0])
+   {
+      // Catch the exception gracefully to handle seeing ";received="
+      try
+      {
+         pb.skipChar();
+         pb.skipWhitespace();
+
+         // Handle cases such as ";received="
+         if(terminators[(unsigned char)(*pb.position())])
+         {
+            ErrLog(<< "Empty value in string-type parameter.");
+            DebugLog(<< "Define p_received with no value");
```

Comment:
What is this DebugLog?  There is already and ErrLog, what purpose does the DebugLog serve?  What is the text of the debug log intending to communicate?  In particular what did you mean by "Define"?

Reply:
Sorry, it was my investigation/testing log. I have removed it.